Source/WebInspectorUI/UserInterface/Views/SingleSidebar.css

```diff
@@ -47,6 +47,16 @@
     border-inline-end: 1px solid var(--border-color);
 }
 
-.single-sidebar.trailing {
+body:not(.narrow) .single-sidebar.trailing {
     border-inline-start: 1px solid var(--border-color);
 }
+
+body.narrow .sidebar.trailing .single-sidebar {
+    width: 100% !important;
+    border-top: none;
+    min-height: auto;
+}
+
+body.narrow .sidebar.trailing .single-sidebar .resizer {
```

Comment:
ditto above about `body.narrow .single-sidebar.trailing > .resizer.vertical-rule`

we'll also want to change the `.resizer` elsewhere in this file to be `.resizer.vertical-rule` too

Reply:
Done. The horizontal rule is still extant on the sub-sidebars though, so I've added a separate CSS rule to always disable it as well.